src/benchmark/RemotePingPong/Program.cs

```diff
@@ -197,6 +199,8 @@ private static async Task<(bool, long, int)> Benchmark(int numberOfClients, long
             var waiting = Task.WhenAll(tasks);
             await Task.WhenAll(waiting);
             sw.Stop();
+            
+            var endThreads = Process.GetCurrentProcess().Threads.Count;
```

Comment:
Collected at the end of the benchmark but before the `ActorSystem`s are terminated.